Sources/eppo/EppoClient.swift

```diff
@@ -1,5 +1,8 @@
 import Foundation;
 
+// Public namespace for Eppo SDK
+public let EPPO_NAMESPACE = "cloud.eppo"
```

Comment:
nit: I believe Swift conventions don't like screaming constants (e.g., see sdk name/version below)

major: probably shouldn't be public

```suggestion
package let eppoNamespace = "cloud.eppo"
```